core/cli/cli.js

```diff
@@ -118,3 +152,45 @@ function green_checkmark() {
     const chalk = require('chalk');
     return chalk.green('\u2714');
 }
+
+async function createScaffold(projectName) {
+    const {homeViewTemplate, homePageTemplate} = require('./templates/homeTemplate');
+    const {jsonPkgTemplate} = require('./templates/jsonPkgTemplate');
+    const viewTemplate = homeViewTemplate();
+    const pageTemplate = homePageTemplate(projectName);
+    const pkgTemplate = jsonPkgTemplate(projectName);
+    let currentDir = path.resolve(process.cwd(), projectName);
+
+    // add files to projectName/app/views
+    let viewPath = path.resolve(currentDir, 'app', 'views');
+    let viewFileName = 'homeView.js';
+    await fs.outputFile(path.resolve(viewPath, viewFileName), viewTemplate);
+
+    // add files to projectName/app/pages
+    let pagePath = path.resolve(currentDir, 'app', 'pages');
+    let pageFileName = 'homePage.html.js';
+    await fs.outputFile(path.resolve(pagePath, pageFileName), pageTemplate);
+
+    // add files to projectName root directory
+    let pkgFileName = 'package.json';
+    await fs.outputFile(path.resolve(currentDir, pkgFileName), pkgTemplate);
+
+    install(currentDir);
+}
+
+function install(directory) {
+    const child = spawn('npm', ['install'], { stdio: 'inherit', cwd: directory });
+
+    child.stdout.on('data', data => {
+        console.log(data);
+    }
+
+    child.on('close', code => {
+        console.log(`process completed with code: ${code}`);
+    }
+}
+
+process.on('unhandledRejection', err => {
```

Comment:
I'd put this at the beginning of the file: We want to listen to rejected promises as soon as the program starts